src/Microsoft.Tye.Hosting/PortAssigner.cs

```diff
@@ -44,12 +44,6 @@ static int GetNextPort()
 
                 foreach (var binding in service.Description.Bindings)
                 {
-                    if (binding.ConnectionString != null)
```

Comment:
Why this change?

Reply:
Ah, found it.

Reply:
yeah, we need to assign ports for bindings even when they have a connection string.